src/Sfa.Tl.ResultsAndCertification.Application/Mappers/AnalystOverallResultExtractionMapper.cs

```diff
@@ -22,7 +22,7 @@ public AnalystOverallResultExtractionMapper()
                 .ForMember(d => d.Status, opts => opts.MapFrom(s => s.Status.ToString()))
                 .ForMember(d => d.Ukprn, opts => opts.MapFrom(s => s.TqProvider.TlProvider.UkPrn))
                 .ForMember(d => d.ProviderName, opts => opts.MapFrom(s => s.TqProvider.TlProvider.Name.Replace(",", " ")))
-                .ForMember(d => d.LastName, opts => opts.MapFrom(s => s.TqRegistrationProfile.Lastname))
```

Comment:
Could we use this  .ForMember(d => d.TlevelTitle, opts => opts.ConvertUsing(new DoubleQuotedStringConverter(), s => s.TqProvider.TqAwardingOrganisation.TlPathway.TlevelTitle))